src/tfr_dumping/src/dumping_action_server.cpp

```diff
@@ -62,7 +63,7 @@ class Dumper
         {
             ROS_INFO("dumping action server initializing");
             detector.waitForServer();
-            aruco.waitForServer();
+            //aruco.waitForServer();
```

Comment:
Why is this call commented out?

Reply:
I remember why it's because of levi's aruco bug